bullet_train-themes/app/views/themes/base/attributes/_has_many.html.erb

```diff
@@ -3,15 +3,18 @@
 <% link_source ||= nil %>
 <% disable_links ||= false %>
 <% link_options ||= {} %>
+<% label_string_method ||= nil %>
 
 <% if object.public_send(attribute).any? %>
   <%= render 'shared/attributes/attribute', object: object, attribute: attribute, strategy: strategy do %>
     <%= object.public_send(attribute).map do |child_object| %>
+      <% label_string = label_string_method ? child_object.public_send(label_string_method) : child_object.label_string %>
```

Comment:
I haven't tested this suggestion, but I think we could consolidate and simplify this a little bit.

```suggestion
<% label_string_method ||= :label_string %>

<% if object.public_send(attribute).any? %>
  <%= render 'shared/attributes/attribute', object: object, attribute: attribute, strategy: strategy do %>
    <%= object.public_send(attribute).map do |child_object| %>
      <% label_string = child_object.public_send(label_string_method) %>
```